### DISCUSSION TITLE
Successful flash of EM7455, no power light after

### DISCUSSION THREAD

**Discussion by ghost** [danielewood/sierra-wireless-modems#173] [General]
I  have an EM7455 and am using an external enclosure with a bright blue power light. Prior to flashing the power light would come on after a bit.

After flashing successfully, the power light only flashes briefly after USB is plugged in and then there is no light after that.

Any ideas?

**Comment by danielewood** [comment]
Sounds like the modem is in/stuck in boothold mode.

What does `lsusb` show?

**Comment by ghost** [comment]
lsusb shows
```
Bus 002 Device 003: ID 0781:5597 SanDisk Corp. 
Bus 002 Device 002: ID 8087:0024 Intel Corp. Integrated Rate Matching Hub
Bus 002 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
Bus 008 Device 002: ID 1058:25fb Western Digital Technologies, Inc. easystore 25FB
Bus 008 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
Bus 007 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
Bus 006 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
Bus 005 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
Bus 001 Device 005: ID 04d9:0169 Holtek Semiconductor, Inc. 
Bus 001 Device 004: ID 046d:c52b Logitech, Inc. Unifying Receiver
Bus 001 Device 003: ID 05e3:0608 Genesys Logic, Inc. Hub
Bus 001 Device 002: ID 8087:0024 Intel Corp. Integrated Rate Matching Hub
Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
Bus 004 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
Bus 003 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
```

running the script again shows the following in loop
```
Could not find any EM7455/MC7455 USB modems
Unplug and reinsert the EM7455/MC7455 USB connector...
```

The USB port on the computer and USB cable are the same ones that were used when flashing with the script. It's housed in this NGFF to USB enclosure:
![image](https://user-images.githubusercontent.com/69406351/114208355-feefd380-9922-11eb-85ba-3c537e4a8bd0.png)

**Comment by danielewood** [comment]
Pretty sure it's dead. 

Run `dmesg -c` to clear the history. 
then re-plug the modem. Wait 30 seconds, and run `dmesg`

If nothing shows up, the modem is definitely dead. It happens. I had a Em7565 just randomly die on me after a month, which was annoying since it was from China and $200 at the time. 

I also had a 7455 die after six months and fry the RBM11G it was in.

**Comment by xikamo** [comment]
Use SED (Smart Error Detection) to solve this issue. Module tracks consecutive resets occuring soon after power-on. After a sixth consecutive reset, the module waits in boot-and-hold mode for a firmware download to resolve the power-cycle problem.
